dashboard/src/components/AppList/__snapshots__/AppList.test.tsx.snap

```diff
@@ -136,6 +136,7 @@ exports[`when custom resources available matches the snapshot 1`] = `
       <div>
         <CardGrid>
           <CustomResourceListItem
+            cluster="defaultc"
```

Comment:
Is this a typo?

Reply:
No - all the tests were already using `default` for the default namespace value, so I wanted to use something different for the default cluster value for comparisons in tests. Perhaps I'll switch to something more explicit `default-cluster` in future tests (and update this one).